vumi_twilio_api/twilio_api.py

```diff
@@ -84,8 +119,145 @@ def usage_exception(self, request, failure):
+                    'Accounts': '/%s/Accounts%s' % (self.version, format_),
+                },
+            },
+        }
         return self._format_response(request, ret, format_)
+
+    @app.route(
+        '/Accounts/<string:account_sid>/Calls',
+        defaults={'format_': ''},
+        methods=['POST'])
+    @app.route(
+        '/Accounts/<string:account_sid>/Calls<string:format_>',
+        methods=['POST'])
+    @inlineCallbacks
+    def make_call(self, request, account_sid, format_):
+        """Making calls endpoint
+        https://www.twilio.com/docs/api/rest/making-calls"""
+        # TODO: Support ApplicationSid field
+        # TODO: Support SendDigits field
+        # TODO: Support IfMachine field
+        # TODO: Support Timeout field
+        # TODO: Support Record field
+        fields = self._validate_make_call_fields(request, format_)
+        fields['AccountSid'] = account_sid
+        fields['CallId'] = self._get_sid()
+        fields['DateCreated'] = self._get_timestamp()
+        fields['Uri'] = '/%s/Accounts/%s/Calls/%s' % (
+            self.version, account_sid, fields['CallId'])
+        message = yield self.vumi_worker.send_to(
+            fields['To'], '',
+            from_addr=fields['From'],
+            session_event=TransportUserMessage.SESSION_NEW,
+            to_addr_type=TransportUserMessage.AT_MSISDN,
+            from_addr_type=TransportUserMessage.AT_MSISDN
+        )
+        yield self.vumi_worker.session_manager.create_session(
+            message['message_id'], **fields)
+        returnValue(self._format_response(request, {
+            'Call': {
+                'Sid': fields['CallId'],
+                'DateCreated': fields['DateCreated'],
+                'DateUpdated': fields['DateCreated'],
+                'ParentCallSid': None,
+                'AccountSid': account_sid,
+                'To': fields['To'],
+                'FormattedTo': fields['To'],
+                'From': fields['From'],
+                'FormattedFrom': fields['From'],
+                'PhoneNumberSid': None,
+                'Status': 'queued',
+                'StartTime': None,
+                'EndTime': None,
+                'Duration': None,
+                'Price': None,
+                'Direction': 'outbound-api',
+                'AnsweredBy': None,
+                'ApiVersion': self.version,
+                'ForwardedFrom': None,
+                'CallerName': None,
+                'Uri': '%s%s' % (fields['Uri'], format_),
+                'SubresourceUris': {
+                    'Notifications': '%s/Notifications%s' % (
+                        fields['Uri'], format_),
+                    'Recordings': '%s/Recordings%s' % (fields['Uri'], format_),
+                }
+            }
+        }, format_))
+
+    def _get_sid(self):
+        return str(uuid.uuid4()).replace('-', '')
+
+    def _get_timestamp(self):
+        return datetime.now(tzlocal()).strftime('%a, %d %b %Y %H:%M:%S %z')
```

Comment:
No real reason, it's just what I'm used to using. We could very well use `tzutc` instead. It's what their API seems to be using (`+0000`)